### ISSUE TITLE
Syntax highlighting for nested elements is broken

### ISSUE THREAD

**Issue by marcofugaro** [csstools/postcss-language#5]
When using nesting, the selectors are highlighted as css properties. They should be highlighted as selectors instead.

| Current behaviour  | Expected behaviour |
| ------------- | ------------- |
| <img width="290" alt="Screenshot 2020-07-15 at 17 29 45" src="https://user-images.githubusercontent.com/7217420/87566049-bde3cf00-c6c2-11ea-8f7d-8149c89d8484.png"> | <img width="346" alt="Screenshot 2020-07-15 at 17 30 20" src="https://user-images.githubusercontent.com/7217420/87566060-c2a88300-c6c2-11ea-9d02-7bd129c7c72a.png"> |

**Comment by jonathantneal** [comment]
@marcofugaro, you are correct. A goal of this language extension is to handle both the syntactically valid possibilities of https://www.w3.org/TR/css-syntax-3/ as well as those achievable in most PostCSS plugins.

As defined in the CSS syntax, your example would produce a faulty `img` declaration (_thus the color coding_) that never ends until the `.wrapper` closes. Reference: https://drafts.csswg.org/css-syntax/#consume-list-of-declarations

However, I believe PostCSS performs a special "look-ahead" to reconsume your code (lines 2-4) as an inner-rule. I think I can and should add this same ability to this language extension. Thanks for bringing this to my attention.

**Comment by Shaker-Hamdi** [comment]
Any updates on this?

**Comment by toddprouty** [comment]
@Shaker-Hamdi, I ended up installing an alternative extension, [PostCSS Intellisense and Highlighting](https://marketplace.visualstudio.com/items?itemName=vunguyentuan.vscode-postcss). It handles nested syntax highlighting and Emmet abbreviations, and I haven't noticed any other issues. 

From the description:

> This extension is a mixed of two famous extensions [postcss-language](https://github.com/csstools/postcss-language.git) and [vscode-postcss-language](https://github.com/MhMadHamster/vscode-postcss-language).

**Comment by SceptreData** [comment]
@toddprouty this is what I ended up doing, albeit with some minor grumbling. As far as I know it uses the scss language server. I don't know enough about any of this to say if that's good or bad but it's probably not ideal!
